BootloaderCorePkg/Include/Library/BootloaderCoreLib.h

```diff
@@ -126,6 +126,22 @@ typedef struct {
   UINT8         BootPartition;
 } S3_DATA;
 
+typedef struct {
+  UINT32  ApicId;
+  UINT32  SmmBase;
+} CPU_SMMBASE;
+
+typedef struct {
+  UINT32       Signature;
+  UINT8        CpuEntry;
+  UINT8        RegType;
+  UINT8        RegWidth;
+  UINT8        Unused;
+  UINT32       SmiEnReg;
+  UINT32       SmiEnVal;
+  CPU_SMMBASE  SmmBase[];
+} SMMBASE_INFO;
```

Comment:
Since LDR_SMM_INFO already has SMI enabling value and address, is it still required to have SmiEnReg, SmiEnVal, RegType and RegWidth?
Should we add Revision field so that later we might add more other fields?

Reply:
LDR_SMM_INFO is the HOB structure from SBL to PLD. SMMBASE_INFO is the communication structure from PLD to SBL. Will add changes to APL platform package in the open source to show how these fields are used.